tables/basic-table/step6.md

```diff
@@ -0,0 +1,44 @@
+In the next few steps, we'll dive into incorporating the Pagination component so users can better control their view of the data.
+
+## Task
+
+1) Import Pagination component from react-core and render it just above the Table component.
+
+<strong>Note: </strong> Group the Table and Pagination together withing a single fragment `<>` container.
+
+<pre class="file" data-target="clipboard">
+import { Pagination } from '@patternfly/react-core';
+</pre>
+
+2) Set the itemCount prop of Pagination to the total number of rows in your dataset
+
+<pre class="file" data-target="clipboard">
+itemCount={defaultRows.length}
+</pre>
+
+At this point you should see the pagination component above the table. Your component should look like the following
+
+<pre class="file">
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { Table, TableHeader, TableBody } from "@patternfly/react-table";
+import { Pagination } from "@patternfly/react-core";
+import { columns, defaultRows } from "./data";
+const BasicTableComponent = () => {
+  return (
+    <>
```

Comment:
Empty tags may not render properly here? I discovered this doesn't work with our examples on patternfly.org, for example -- gatsby is fine . May want to use React.Fragment instead.

We use react-live with Gastby, which uses https://github.com/bublejs/buble and allows a custom jsxFragment.